lib/data/demos.dart

```diff
@@ -41,18 +41,22 @@ enum GalleryDemoCategory {
   study,
   material,
   cupertino,
-  other,
+  other;
+
+  @override
+  String toString() {
+    return name.toUpperCase();
+  }
 }
 
 extension GalleryDemoExtension on GalleryDemoCategory {
```

Comment:
Hi, happy to have you contribute! 

Enhanced enums allow us to get rid of this extension, altogether. Can we move displayTitle inside the enum?

Reply:
Hey, yes sure i fixed this in the new commit